projects/templates/projects/upload_project.html

```diff
@@ -0,0 +1,46 @@
+{% extends "codeeve/index.html" %}
+
+{% block header %}
+{% endblock %}
+
+{% block content %}
+    <div class="content-section-a">
+
+        <div class="container">
+            <div class="row">
+                <div class="col-lg-5 col-sm-6">
+                    <hr class="section-heading-spacer">
+                    <div class="clearfix"></div>
+                    <h2 class="section-heading">{{ project.title }}</h2>
+    <form action="/projects/" method="post">
+        {% csrf_token %}
+        {{ form.title.label }}: {{ form.title }}
+        <br>
+        {{ form.description.label }}: {{ form.description }}
+        <br>
+        {{ form.max_members.label }}: {{ form.max_members}}
+        <br>
+        {{ form.difficulty.label }}: {{ form.difficulty }}
+        <br>
+        {{ form.owner.label }}: {{ form.owner }}
+        <br>
+        {{ form.participants.label }}: {{ form.participants }}
```

Comment:
why not simply `{{form}} and play with the fields in the UploadForm class? It reduces repetition and typos